.github/workflows/main.yml

```diff
@@ -12,12 +12,8 @@ jobs:
       fail-fast: false
       matrix:
         os: [windows, ubuntu, macos]
-        python: ["3.6", "3.7", "3.8", "3.9", "3.10"]
+        python: ["3.7", "3.8", "3.9", "3.10"]
```

Comment:
Is there any particular reason to do this now?

Reply:
It [EOLs in 3 days](https://endoflife.date/python). Does not have to be a part of this PR, just thought I'd clean up the matrix. Considering EOL, IMHO we should stop supporting it in 3.0.

Reply:
I don't see a compelling reason to remove it now, though. If we see it acting up, we can remove support, but that means (to me) that we also adjust the package metadata and remove the interop file. Please revert this.

Reply:
@filmor OK, moved into https://github.com/pythonnet/pythonnet/issues/1640